python/svix/api.py

```diff
@@ -61,7 +51,7 @@ def __init__(self, configuration: Configuration) -> None:
     @contextmanager
     def _api(self) -> t.Generator[ApiClass, None, None]:
         with ApiClient(self._configuration) as api_client:
-            yield self._ApiClass(api_client)
+            yield t.cast(t.Any, self._ApiClass)(api_client)
```

Comment:
Why do we need to cast this to Any?

Reply:
Because the OpenAPI generator APIs are not all inheriting from the same type (but all share the same structure). Don't ask me why.
This cast is a way to force it to work. Let me try and see if I can fix it in a less ugly way.

Reply:
I made it less casty, but still with a cast that I couldn't figure out.